packages/skill-lesson/lib/modules.ts

```diff
@@ -184,7 +184,7 @@ export const getModuleById = async (id: string) =>
         _type,
         title,
         "slug": slug.current,
-        "image": image.asset->url,
+        "image": image.url,
```

Comment:
for epic react, this was the structure needed to get the module image. Making a change to a monorepo package makes me a little nervous though